dojo/importers/reimporter/reimporter.py

```diff
@@ -124,6 +134,15 @@ def process_parsed_findings(self, test, parsed_findings, scan_type, user, active
                 else:
                     # existing findings may be from before we had component_name/version fields
                     logger.debug('%i: updating existing finding: %i:%s:%s:%s', i, finding.id, finding, finding.component_name, finding.component_version)
+                    if not (finding.mitigated and finding.is_mitigated):
+                        logger.debug('Item matches a finding that is currently open.')
+                        if item.mitigated:
+                            logger.debug('%i: closing: %i:%s:%s:%s', i, finding.id, finding, finding.component_name, finding.component_version)
+                            finding.mitigated = item.mitigated
+                            finding.is_mitigated = True
+                            finding.mitigated_by = None
```

Comment:
Why do you set `finding.mitigated_by` to `None`? Maybe the attribute is set in `item`.